.github/workflows/velox_docker.yml

```diff
@@ -532,7 +532,7 @@ jobs:
       fail-fast: false
       matrix:
         spark: [ "spark-3.2" ]
-        celeborn: [ "celeborn-0.4.1", "celeborn-0.3.2-incubating" ]
+        celeborn: [ "celeborn-0.5.0", "celeborn-0.4.1", "celeborn-0.3.2-incubating" ]
```

Comment:
The workflow [requires a download link based on the specific version](https://github.com/apache/incubator-gluten/pull/6264/files#diff-38eda6989d285e6a3d79a3a66cc656d0b6d483b3d5c77bd0969d878642fd0f0eR573-L576 ) to deploy the celeborn service, then we should have to use the version instead of profile id.

Reply:
@yikf, I see. Thanks!